extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -134,3 +134,16 @@ export function doubleclickIsA4AEnabled(win, element) {
   }
   return enableA4A;
 }
+
+export const adConfig = {
```

Comment:
renderStart is not used when we do preferential rendering, but we can have a case where fast fetch is used, but it ends up failing client side validation and falling back to using XOriginIframeHandler, in which case renderStart does apply